src/dotnet/APIView/APIViewWeb/Client/src/comments.ts

```diff
@@ -90,6 +90,44 @@
         e.preventDefault();
     });
 
+    $(document).on("click", ".js-github", e => {
+        let target = $(e.target);
+        let repo = target.attr("data-repo");
+        let language = getLanguage(e.target);
+
+        // Fall back to the repo-language if we don't know the review language,
+        // e.g.Go doesn't specify Language yet.
+        if (!language) {
+          language = target.attr("data-repo-language");
+        }
+        let commentElement = getCommentElement(getCommentId(e.target)!);
+        let comment = commentElement.find(".js-comment-raw").html();
+        let codeLine = commentElement.closest(".comment-row").prev(".code-line").find(".code");
+        let apiViewUrl = "";
+
+        // if creating issue from the convos tab, the link to the code element is in the DOM already.
+        let commentUrlElem = codeLine.find(".comment-url");
+        if (commentUrlElem.length) {
```

Comment:
Would the logic work if we remove this branch?

Reply:
This is to handle adding comment url from convos or review tab.

Reply:
Why doesn't `https://github.com/Azure/azure-sdk-tools/pull/1023/files/ee3dcb2f91ceaeab110e9c8fae95b9cdd5978c5b#diff-640ab677b570ba8bd56867459d622b7eR116 work there?

Reply:
Well, for one thing the path from window.location.href would be wrong as we wouldn't know what revision the comment is from.

Reply:
OK